src/baseline.rs

```diff
@@ -247,27 +283,29 @@ impl BaselineLoader for RustdocBaseline {
 }
 
 pub(crate) struct PathBaseline {
+    target_root: PathBuf,
     root: PathBuf,
-    lookup: std::collections::HashMap<String, (String, PathBuf)>,
+    lookup: std::collections::HashMap<String, Manifest>,
 }
 
 impl PathBaseline {
-    pub(crate) fn new(root: &std::path::Path) -> anyhow::Result<Self> {
+    pub(crate) fn new(
+        target_root: &std::path::Path,
+        root: &std::path::Path,
+    ) -> anyhow::Result<Self> {
```

Comment:
Please document these parameters. When there was one parameter, it was decently clear what it did. Now that there are two parameters of the same type, it isn't obvious which one does what.

Also, consider whether this order or the opposite order is better and less likely to lead to bugs.

Reply:
> Also, consider whether this order or the opposite order is better and less likely to lead to bugs.

I based the order on `RegistryBaseline`. From what I understand, you suggest that the "more important" argument should be first. Then, should I also change the order in `RegistryBaseline`?

Reply:
Done in 5ba44475ffc0e8a4e7003327a823acad7d1890c9. Is that what you had in mind?